index.ts

```diff
@@ -47,6 +48,16 @@ const exit_handler = async () => {
 
 	if (config.DEBUG) { app.use(express.static('web/dist/static')); }
 
+	app.use(cookie_parser());
+
+	app.get('/user_problems', async (req, res) => {
+		if (Object.prototype.hasOwnProperty.call(req.cookies, 'user_string')) {
+			res.send(await database.get_user_problems(req.cookies.user_string));
+		} else {
+			res.send([ 1, ]);
```

Comment:
Does there need to be a comma here?

Reply:
Yes it's a trailing comma and makes diffs nicer. Required by eslint.